app/_hub/kong-inc/grpc-web/index.md

```diff
@@ -0,0 +1,158 @@
+---
+name: gRPC-Web
+publisher: Kong Inc.
+
+categories:
+  - transformations
+
+type: plugin
+
+desc: Allow browser clients to call gRPC services.
```

Comment:
This can be more accurate. Maybe "Allow HTTP clients (such as browsers) to access gRPC services" is better? The plugin can definitely be used by non-browsers as well.

Reply:
Sure, gRPC are already accessible by HTTP(2) clients.  But gRPC-Web was designed specifically around the limitations of JS' Fetch API, and the gRPC-Web library is mainly for browsers.  (I guess it can be used on Node.js, but not without some extra work).

This plugin's extension to use `application/json` content is non-standard and actually eschews the gRPC-Web protocol, which is why it has some extra limitations around streaming I/O.